exp/agents/mrkl/prompt.go

```diff
@@ -0,0 +1,101 @@
+package mrkl
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/tmc/langchaingo/exp/prompts"
+	"github.com/tmc/langchaingo/exp/tools"
+)
+
+const PREFIX = `Answer the following questions as best you can. You have access to the following Tools:`
```

Comment:
prefer a `const (` expression,

Do not do full-caps exports of constants, instead use TitleCase (and comment the export) or lowerCamel.